frontend/ui/widget/configdialog.lua

```diff
@@ -180,18 +185,38 @@ function ConfigOption:init()
             local option_height = Screen:scaleBySize(self.options[c].height and self.options[c].height or default_option_height)
             local item_spacing_width = Screen:scaleBySize(self.options[c].spacing and self.options[c].spacing or default_items_spacing)
```

Comment:
It's probably best to add a `local padding = Size.padding.small` to prevent repeated table lookups. It's not nearly as expensive as `Font:getFace` but still. :-)